### PR TITLE
Release 1.1 RC 2

### PR DESCRIPTION
**Pull request by Szymongib**
**Description**

Changes proposed in this pull request:

- Create RC 2

**Related issue(s)**

### CODE REVIEW

File: resources/core/values.yaml [hunk 1]
```diff
@@ -24,65 +24,65 @@ global:
     secretName: etcd-backup-abs-credentials
   acceptance_tests:
     dir:
-    version: 1.1.0-rc1
+    version: 1.1.0-rc2
   ui_acceptance_tests:
     dir:
     version: "83aa639b"
```
Comment (strekm) [inline resources/core/values.yaml]: is this on purpose or should it be `1.1.0-rc2`?

File: resources/helm-broker/values.yaml [hunk 2]
```diff
@@ -32,10 +32,10 @@ global:
...
     dir:
-    version: 1.1.0-rc1
+    version: 1.1.0-rc2
   alpine_net:
     dir:
-    version: 1.1.0-rc1
+    version: 1.1.0-rc2
   add_ons_ui:
     dir: ""
     version: 23b3d5a1
```
Comment (strekm) [inline resources/helm-broker/values.yaml]: this one also looks suspicious. please verify

File: resources/core/values.yaml (hunk 1 above)
Comment (Szymongib) [reply resources/core/values.yaml]: It is on purpose, `ui_acceptance_tests` is part of a console, which is not being released.

File: resources/helm-broker/values.yaml (hunk 2 above)
Comment (Szymongib) [reply resources/helm-broker/values.yaml]: `add_ons_ui` also is not being released.

**Pull request merged**